tools/roslaunch/src/roslaunch/substitution_args.py

```diff
@@ -163,8 +163,7 @@ def _find_executable(resolved, a, args, _context):
     # we try to find the specific executable in libexec via catkin
     # which will search in install/devel space
     full_path = None
-    # don't know why not imported at the top
-    from roslaunch.resource_cache import find_in_workspaces
+    from catkin.find_in_workspaces import find_in_workspaces
     paths = find_in_workspaces(['libexec'], project=args[0], first_matching_workspace_only=True)  # implicitly first_match_only=True
```

Comment:
I have added it in fa79ca8c23c9f0ebcdb197d6848e6c7d341a4883 but I don't have any launch file to exercise this and confirm the performance improvement. Can you please point me to a use case which benefits from this change?